txnkv/transaction/txn.go

```diff
@@ -638,6 +665,34 @@ func (txn *KVTxn) InitPipelinedMemDB() error {
 	return nil
 }
 
+func (txn *KVTxn) throttle() {
```

Comment:
What about naming the function with "p-txn"?

```suggestion
func (txn *KVTxn) pipelinedTxnThrottle() {
```